app/models/service_template_embedded_terraform_template.rb

```diff
@@ -0,0 +1,9 @@
+class ServiceTemplateEmbeddedTerraformTemplate < ServiceTemplate
+  def self.default_provisioning_entry_point(_service_type)
+    '/Service/Generic/StateMachines/GenericLifecycle/provision'
+  end
+
+  def self.validate_config_info(options)
+    options
```

Comment:
the other service templates (orchestration, ansible playbook, awx, etc.) all seem to return the value of the key `:config_info` in options:

```ruby
  def self.validate_config_info(options)
    config_info = options[:config_info]
    unless config_info[:configuration_script_id] || config_info[:configuration]
      raise _('Must provide configuration_script_id or configuration')
    end

    config_info
  end
```

Should we do that here?

Reply:
This is just a placeholder until I understood what the value of options was, service_template_ansible_playbook and service_template_orchestration/service_template_container_template handle this differently so I want to see where the options come from.

Reply:
yeah, i honestly have no idea either